server/src/main/java/com/linecorp/centraldogma/server/metadata/Token.java

```diff
@@ -64,8 +64,11 @@ public final class Token implements Identifiable {
     @Nullable
     private final UserAndTimestamp deactivation;
 
+    @Nullable
+    private final UserAndTimestamp deletion;
```

Comment:
As it stands, deletion is possible even if a token is not deactivated.

We should also add `deletion` to the condition for determining whether a token is active

```
public boolean isActive() {
    return deactivation == null && deletion == null;
}
```